logs/ssl.rst

```diff
@@ -358,6 +358,48 @@ practitioners concerned by the loss of visibility, and the opportunity for
 intruders to leverage ESNI-enabled servers and Doh-enabled clients to evade
 inspection.
 
+
+QUIC
+====
+
+.. versionadded:: 6.1
+
+The QUIC protocol integrates encryption, stream multiplexing and flow control
+at the transport layer. QUIC uses TLS 1.3 by default. The QUIC analyzer outputs
+logs to :file:`ssl.log`. The QUIC analyzer attempts to handle the INITIAL packet
+payload to provide greater observability.
```

Comment:
Thanks for the PR - I'd think a small `logs/quic.rst` file, noting that `ssl.log` entries are created as well would better fitting.

There's a separate `quic.log`. Anything showing in `ssl.log` is because the QUIC analyzer forwards data to the SSL analyzer, not because it explicitly writes to the `ssl.log`.